SPARCore/GADArticle.m

```diff
@@ -17,7 +59,7 @@ @implementation GADArticle
         article.tags = [NSArray arrayWithObjects:@"amazing!", @"fantastic!", @"Computer science!", nil];
         article.brief = [NSString stringWithFormat:@"This is very brief %i", i];
         article.content = @"This is much less brief because it should be longer than the brief";
-        article.datePublished = [NSDate date];
+        //article.datePublished = [NSDate date];
```

Comment:
At the time, yes. It was because I wasn't sure how we were going to handle the datePublished field (keep it an NSString or turn it into an NSDate) but now that it's clearly going to be an NSDate this should no longer be an issue and can be uncommented.